src/themes/masschallenge/elements/container.overrides

```diff
@@ -0,0 +1,64 @@
+/*******************************
+         Theme Overrides
+*******************************/
+
+.container > .grid.primary {
+        display: grid;
+        grid-template-areas: "topbar" "content" "footer" "mobile-tab-nav";
+        grid-template-rows: 55px 1fr 58px;  
```

Comment:
I realized after the last commit that explicitly defining the row height for the third row causes some minor formatting issues. Suggested change:
`grid-template-rows: 55px 1fr auto;`

**More info needed here. Will update this comment.**